drivers/gles3/storage/texture_storage.cpp

```diff
@@ -1842,8 +1898,10 @@ void TextureStorage::_clear_render_target(RenderTarget *rt) {
 	if (rt->backbuffer_fbo != 0) {
 		glDeleteFramebuffers(1, &rt->backbuffer_fbo);
 		glDeleteTextures(1, &rt->backbuffer);
+		glDeleteTextures(1, &rt->backbuffer_depth);
```

Comment:
As this could be either color buffer, or depth buffer, or both, we probably need to check the value before we delete.

Reply:
Added a separate check if the backbuffer depth is defined in the delete section